erpnext/vehicles/report/vehicle_maintenance_schedule/vehicle_maintenance_schedule.js

```diff
@@ -90,5 +90,41 @@ frappe.query_reports["Vehicle Maintenance Schedule"] = {
 			fieldtype: "Link",
 			options: "Item Group"
 		},
-	]
-};
\ No newline at end of file
+	],
+
+	onChange: function(new_value, column, data, rowIndex) {
+		if (column.fieldname == "remarks") {
+			if (cstr(data['remarks']) === cstr(new_value)) {
+				return
+			}
+
+			if (!data.opportunity) {
+				frappe.msgprint(__("Opportunity does not exist"));
+				frappe.run_serially([
```

Comment:
No need for `run_serially` since this method does not return a promise and neither does `query_report_local_refresh` return a promise and neither does `frappe.query_report.datatable.bodyRenderer.render();`

Reply:
something in the scope of somewhere is being suppressed/throttled. Tried to do it without run_serially but the value doesn't change